pulse_actions/run_time_config.json

```diff
@@ -1,7 +1,7 @@
 {
-    "buildbot": {
+    "manual_backfill": {
         "exchange": "exchange/treeherder/v1/job-actions",
-        "topic": "buildbot.#.#"
+        "topic": "buildbot.#.backfill"
     },
```

Comment:
I think since we only support backfill right now, there is no use to listen to all the 'retrigger' and 'cancel' events. It unnecessarily increases our queue.